### ISSUE TITLE
[4.0] Help button opens new window

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#22544]
The help button on the toolbar opens in a new window. To satisfy the accessibility rules there should be both a visible and audible notification of this.

There is no option to enable passing the required notification to the button

**Comment by bembelimen** [comment]
Do you have a link/code snippet how this should look like?

**Comment by brianteeman** [comment]
For users using screenreaders this could be addressed by adding the text inside a sr-only span
```
<joomla-toolbar-button id="toolbar-help">
<button class="button-help btn btn-info" type="button" onclick="Joomla.popupWindow('https://help.joomla.org/proxy?keyref=Help40:Content_Article_Manager&amp;lang=en', 'Help', 700, 500, 1)">
	<span class="fa fa-question" aria-hidden="true"></span>
	Help
	<span class="sr-only">Opens in a new window</span>
</button>
</joomla-toolbar-button>
```

For sighted users we can simply use the title attribute so the full markup would be

```
<joomla-toolbar-button id="toolbar-help">
<button class="button-help btn btn-info" type="button" onclick="Joomla.popupWindow('https://help.joomla.org/proxy?keyref=Help40:Content_Article_Manager&amp;lang=en', 'Help', 700, 500, 1)"
title="Opens in a new window">
	<span class="fa fa-question" aria-hidden="true"></span>
	Help
	<span class="sr-only">Opens in a new window</span>
</button>
</joomla-toolbar-button>
```

**Comment by brianteeman** [comment]
Instead of using the title we could use the fa-external-link icon like on the admin login screen but in this case I think it would not be clear

**Comment by hardik-codes** [comment]
@brianteeman in which file should the above code has to go ?

**Comment by brianteeman** [comment]
If I knew that I would have done it myself :)

**Comment by bembelimen** [comment]
The output is here: https://github.com/joomla/joomla-cms/blob/4.0-dev/layouts/joomla/toolbar/basic.php

**Comment by hardik-codes** [comment]
Is there any thing remains to be changed @bembelimen as I think the above file already has the code as suggested by @brianteeman

**Comment by brianteeman** [comment]
@hardik-codes no it doesn't. if it did then there wouldnt be an issue to fix :)

**Comment by hardik-codes** [comment]
Working on it

**Comment by ghost** [comment]
closed as having Pull Request #24138

**Comment by ghost** [comment]
Closed as having Pull Request #25519
